composer.json

```
@@ -24,32 +24,32 @@
         }
     ],
     "require": {
-        "php": "^7.2 || ^8.0",
+        "php": "^7.3 || ^8.0",
         "composer-runtime-api": "^2",
-        "doctrine/dbal": "^2.11 || ^3.0",
+        "doctrine/dbal": "^3.0",
         "doctrine/deprecations": "^0.5.3",
         "doctrine/event-manager": "^1.0",
         "friendsofphp/proxy-manager-lts": "^1.0",
         "psr/log": "^1.1.3 || ^2 || ^3",
-        "symfony/console": "^3.4 || ^4.4.16 || ^5.0 || ^6.0",
-        "symfony/stopwatch": "^3.4 || ^4.0 || ^5.0 || ^6.0"
+        "symfony/console": "^4.4.16 || ^5.4 || ^6.0",
+        "symfony/stopwatch": "^4.4 || ^5.4 || ^6.0"
     },
     "require-dev": {
         "ext-pdo_sqlite": "*",
         "doctrine/coding-standard": "^9",
         "doctrine/orm": "^2.6",
-        "doctrine/persistence": "^1.3 || ^2.0",
+        "doctrine/persistence": "^2.0",
         "doctrine/sql-formatter": "^1.0",
         "ergebnis/composer-normalize": "^2.9",
         "phpstan/phpstan": "^1.5",
         "phpstan/phpstan-deprecation-rules": "^1",
         "phpstan/phpstan-phpunit": "^1.1",
         "phpstan/phpstan-strict-rules": "^1.1",
         "phpstan/phpstan-symfony": "^1.1",
-        "phpunit/phpunit": "^8.5 || ^9.4",
-        "symfony/cache": "^3.4.26 || ^4.2.12 || ^5.0 || ^6.0",
-        "symfony/process": "^3.4 || ^4.0 || ^5.0 || ^6.0",
-        "symfony/yaml": "^3.4 || ^4.0 || ^5.0 || ^6.0"
+        "phpunit/phpunit": "^9.4",
```

```suggestion
        "phpunit/phpunit": "^9.5",
```

PHPUnit 9.4 is dead and buried.